src/spinner.vue

```diff
@@ -1,23 +1,24 @@
 <template lang="jade">
-.mdl-spinner.mdl-js-spinner(v-bind:class='{ "mdl-spinner--single-color": singleColor, "is-active": active }')
+.mdl-spinner.mdl-js-spinner(v-bind:class="{ 'mdl-spinner--single-color': singleColor, 'is-active': active, 'is-upgraded': upgraded }")
```

Comment:
I'll review this in more detail. The fact that the class is not added by MDL may be a problem.
Anyway, can you remove it from the data and class, please?